test/e2e/storage/persistent_volumes-gce.go

```diff
@@ -130,7 +131,7 @@ var _ = utils.SIGDescribe("PersistentVolumes GCEPD [Feature:StorageProvider]", f
 
 		ginkgo.By("Deleting the Claim")
 		framework.ExpectNoError(e2epv.DeletePersistentVolumeClaim(ctx, c, pvc.Name, ns), "Unable to delete PVC ", pvc.Name)
-		framework.ExpectEqual(verifyGCEDiskAttached(diskName, node), true)
+		gomega.Expect(verifyGCEDiskAttached(diskName, node)).To(gomega.BeTrue())
```

Comment:
Can you change this to not use BeTrue? See https://github.com/kubernetes/kubernetes/issues/105678.